src/libaktualizr/primary/aktualizr.h

```diff
@@ -82,6 +82,19 @@ class Aktualizr {
    */
   std::future<result::Download> Download(const std::vector<Uptane::Target>& updates);
 
+  /**
+   * Get log of installations. The log is indexed for every ECU and contains
+   * every change of versions ordered by time. It may contain duplicates in
+   * case of rollbacks.
```

Comment:
Is this still true? Won't anything rolled back erase the prior generation number for that target?

Reply:
What you describe is close to the old behavior.
It will now create a new generation and have another entry in the table.